test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -253,6 +254,8 @@ def __init__(self, path, analyze_arg_spec=False, collection=None, base_branch=No
 
         self._python_module_override = False
 
+        self.routing = None
```

Comment:
```suggestion
        self.routing = {}
```
This makes the other code simpler.